x/observer/migrations/v9/migrate.go

```diff
@@ -0,0 +1,38 @@
+package v9
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+type observerKeeper interface {
+	GetBallotListForHeight(ctx sdk.Context, height int64) (val types.BallotListForHeight, found bool)
+	DeleteBallot(ctx sdk.Context, index string)
+	DeleteBallotListForHeight(ctx sdk.Context, height int64)
+}
+
+const MaturityBlocks = int64(100)
```

Comment:
Should we provide maturityBlocks as an arg here?

Reply:
The maturity blocks are part of the emissions params, which makes it challenging to use the parameter. 
I think using the constant here is fine for now

Reply:
> The maturity blocks are part of the emissions params

Why emission and not observer / cross-chain ?

Reply:
Its legacy code ,and should be changed , 
But not in this PR